packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -48,6 +48,8 @@
   "scripts": {
     "vscode:prepublish": "npm prune --production",
     "vscode:package": "vsce package",
+    "vscode:sha256": "node ../../scripts/generate-sha256.js >> ../../SHA256",
```

Comment:
@lcampos , @DatGuyJonathan - This will be published the next time we do a release, which will be on r2b.